zebrad/tests/acceptance.rs

```diff
@@ -1473,3 +1475,81 @@ where
89010000000000000000000000000000000000000000000000000000000000000000ffffffff250340420f0fe4b883e5bda9e7a59ee4bb99e9b1bc104d696e656420627920676176696e313149b7525402249ccd1d000000001976a9149ef3382375369c5f3b5e41ed4fc48fa8549aac3988ac405973070000000017a91447185bab99466b849d774faa2edfa983bd3774148700228c200000000000000000000000000000000400008085202f8901a0960e24269869fe0cb8e81ff6188fdd675d3846b1218e42232cde314b7a9316000000006a4730440220691d6d926a417faf8bd241e02be6ac512c6ab263cf443e1aa64b4cfd553a802702204ac1a0d97030a689427fb272023a6b4477b7f23ca7be4e099430ffde78be1ee70121023c98e6ddf69091291f1eec341a1f5f8364efab0cb2ec5631ab5c2c6822ee4b23ffffffff02111ef205000000001976a91432e79f48ae5aef9c7b60801e1b55428e168eb15c88ac5db90300000000001976a914f45dab2d81f6ea3e1e6050326d2fc02885d7752788ac000000000000000000000000000000000000000400008085202f8901e166bf18d23f6a581391d33472db0202a0f362565fd741b177c1e542d871658a000000006a473044022009d66f79793012ef1201f2a39405979b5c99f60c0eda8fafc7733a463c8d74670220243c4891ca5b6924291513f6bb265ddce132789212de54acb8da31fb7fd2d788012102abeb1ac05121f69df5715a7c4980be3c570c56b4a58fee04651eddb88742701cfeffffff021b9a1755530000001976a914d97325c00fc728bbd0a6ecd84ac6472414d8610588acb2945d81020000001976a9140098a1ad96d580e8ecda1273051385cfe033935988ac35420f0068420f0000000000000000000000000400008085202f89029430018466c24c99f84657a5f000246b9b6cfef54da53c279c491d31e9657b2e5d0000006a473044022070bead6f51c69c09f12dd6921ca84c6fcf649b4ffc74bc58ec186efda1754835022004f71f95383087ea1d286a4d29f91d687194deb8ed3f1e778e05fb8f6512883e012103c21cf6f0ca752e549bb97ca6605ddafa2260b2767e06ebeaa1816e07710e2efdffffffffe16a914d4cd8adb965300ea7e995cfd32bf8ca56821eda63cf419a52284149a9000000006a4730440220612ca09675aa9b57ed35ae232c8fc431dbdb793b0bb866dfcf206b8f2ffa29d902203e71ce227184eaef66632c6f4b079408407ee763445ae57914f809c1874ac4fc0121037e609ec336382fc08dfc98a6568a82eb5b60b4ec324d79bfe12d4ca3949d34bbffffffff014893b5b5040000001976a91457787bcbe709bf4511a79eb356c1772d7a734d5388ac0000000067420f0000000000000000000000000400008085202f890173f77790296444883bd612c94a010d42e1f55f57c11bdf5e36de21b3a3c56936010000006a47304402203c1f53208a3bee1cc69a653da49a9cf13bb4dbddbd9b477300339fab5c68939f02201e69969a553e82dd185d3bf316ceb65240a903d09968e9c82a2d1a33549400cb0121035148109f816ec889ead8066829ad8132f06caaf98db6fdb745452489544336bdffffffff0240420f00000000001976a914ba4ff7e3ee85d17077ec10922fcf3a7cec1d55bb88acbe325207000000001976a91454b42d78507235610da54a3c7a55e942830e41d788ac00000000ff64cd1d00000000000000000000000400008085202f890150474bef0b34199d220b7d7e75a288bcf0cbef3022e287be79aaeae256c4b2ea010000006a473044022004fc4572a019a144d9bcdf795498401d65a453b1031bdf6293b6ea9085f2819802200ad4315170aaa80589254fc66cb5b5280ad9c385d3bb2a2de23d31788555ba7301210224c5d6e287b4f193fe48e874f0ce3ce7e0659c68effc6773502ee726b68a6a56ffffffff0232a00600000000001976a914200f6fae96ed4bebbaf115ddb10228fceeb6ccf588acaeb41c05000000001976a9144554bb3b9c3653c588d9136cdcfe254260be742a88ac0000000068420f000000000000000000000000";
```

Comment:
Performance/Code Size: 2kB is a lot of data to add for a single test, and it might slow down some editors.

Can we parse the height from the block data, or compare its data to an existing block test vector's data?

Reply:
Good call, I changed to a block in MAINNET_BLOCKS, in c9e477a62